docs/instalation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Instalation"
+date: 2020-08-26T10:56:03
+weight: 3
+geekdocRepo: https://github.com/owncloud/file-picker
+geekdocEditPath: edit/master/docs
+geekdocFilePath: instalation.md
+---
+
+{{< toc >}}
+
+## Setup authorization
+The config for authorization is provided via a json file located at the root URL address of your web application which has to be named `file-picker-config.json` and needs to point to your authorization provider. You can take a look at the following example:
+
+```json
+{
+  "server": "<owncloud-server>",
```

Comment:
so the file picker is limited to a single server, there is probably no way of providing a use case where one asks "please enter your ownCloud URL" and then let them connect a website to an arbitrary ownCloud instance, not a pre-configured one. I guess this could be a different user story to look at later, if even technically possible due to the complex auth requirements.

Reply:
Yep, not possible at the moment. But sounds awesome and would be great to try it out later. Let's open a ticket and collect user stories?

Reply:
raised https://github.com/owncloud/file-picker/issues/17